src/main/java/org/ibp/api/rest/germplasm/GermplasmResource.java

```diff
@@ -161,10 +162,11 @@ public long getFilteredCount() {
 
 	@RequestMapping(value = "/crops/{cropName}/germplasm/templates/xls", method = RequestMethod.GET)
 	public ResponseEntity<FileSystemResource> getImportGermplasmExcelTemplate(@PathVariable final String cropName,
-		@RequestParam(required = false) final String programUUID) {
+		@RequestParam(required = false) final String programUUID,
+		@RequestParam(required = false) final boolean updateFormat) {
```

Comment:
Rename as `isGermplasmUpdateFormat`

Reply:
Are we using the same recourse for both templates? seems a bit weird to me. If that's the case, then we can add a PathVariable and witch using the path variable